src/routes/e2e/widget/inline/+page.svelte

```diff
@@ -32,28 +40,28 @@
       const response = await fetch(`${window.location.origin}/api/locale`);
       content = response.ok && (await response.json());
     }
-    // TODO: Add method to refresh form
-    new Widget({
-      target: formEl,
-      props: {
-        config: {
-          clientId: 'WebOAuthClient',
-          redirectUri: `${window.location.origin}/callback`,
-          scope: 'openid profile email me.read',
-          serverConfig: {
-            baseUrl: 'https://openam-crbrl-01.forgeblocks.com/am/',
-            timeout: 5000,
-          },
-          realmPath: 'alpha',
-        },
-        content,
-        links: {
-          termsAndConditions: 'https://www.forgerock.com/terms',
+
+    config.set({
+      config: {
+        clientId: 'WebOAuthClient',
+        redirectUri: `${window.location.origin}/callback`,
+        scope: 'openid profile email me.read',
+        serverConfig: {
+          baseUrl: 'https://openam-crbrl-01.forgeblocks.com/am/',
+          timeout: 5000,
         },
+        realmPath: 'alpha',
+      },
+      content,
+      links: {
+        termsAndConditions: 'https://www.forgerock.com/terms',
       },
     });
+
+    new Widget({ target: formEl, props: { type: 'inline' }});
```

Comment:
This is now the way the dev specifies the form factor. It defaults to modal if `type` isn't given.